x-pack/plugins/reporting/server/usage/types.ts

```diff
@@ -251,3 +251,30 @@ export type ReportingUsageType = RangeStats & {
 };
 
 export type FeatureAvailabilityMap = Record<string, boolean>;
+
+export enum EventType {
+  REPORT_CREATION = 'report_creation',
+  REPORT_CLAIM = 'report_claim',
+  REPORT_COMPLETION_CSV = 'report_completion_csv',
+  REPORT_COMPLETION_SCREENSHOT = 'report_completion_screenshot',
+  REPORT_ERROR = 'report_error',
+  REPORT_DOWNLOAD = 'report_download',
+  REPORT_DELETION = 'report_deletion',
+}
+
+export enum FieldType {
+  REPORT_ID = 'report_id',
+  EXPORT_TYPE = 'export_type',
+  OBJECT_TYPE = 'object_type',
+  IS_DEPRECATED = 'is_deprecated',
+  IS_PUBLIC_API = 'is_public_api',
+  DURATION = 'duration_ms',
```

Comment:
nit: maybe for consistency, all other labels match word-for-word:

```suggestion
  DURATION_MS = 'duration_ms',
```